README.md

```diff
@@ -1,36 +1,31 @@
 # Telescope.nvim
 [![Gitter](https://badges.gitter.im/nvim-telescope/community.svg)](https://gitter.im/nvim-telescope/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-> **Telescope**
-> An arrangement of lenses or mirrors or both that gathers light,
-> permitting direct observation or photographic recording of distant objects.
->
-> — thefreedictionary
-
-![by @glepnir](https://user-images.githubusercontent.com/41671631/100819597-6f737900-3487-11eb-8621-37ec1ffabe4b.gif)
+Gaze deeply into unknown regions using the power of the moon.
 
+## What Is Telescope?
 
-`Telescope.nvim` is a next generation library for creating floating pickers
-with advanced features.  It is written in `lua` and is built on top of latest
-awesome features from `neovim` core.  `Telescope.nvim` is centered around
-modularity so much so that each picker is its own world, (meaning it can be
-configured in isolation from other pickers, such presentation, algorithm,
-mappings ... etc).
+`telescope.nvim` is a highly extendable fuzzy finder over lists.   Built on top of latest
+awesome features from `neovim` core.  Telescope is centered around
+modularity, allowing for easy customization.
 
-`Telescope.nvim` was built with the intention of becoming a library, but it has long grown to
-be much bigger than that.  In addition to
-[extensions](https://github.com/nvim-telescope) `Telescope.nvim` comes with a
-growing number of community driven [built-in pickers](#built-in-pickers),
-covering a wide range of use cases and tools, and offers a customizable user
-interface.
+![by @glepnir](https://user-images.githubusercontent.com/41671631/100819597-6f737900-3487-11eb-8621-37ec1ffabe4b.gif)
 
+`telescope.nvim` has a number of [extensions](https://github.com/nvim-telescope) for supporting specific plugins. 
```

Comment:
Builtin picker should be mentioned before the external extensions.

Reply:
I get that added!